docs/user_docs/metrics/technicalFork/technical_fork.md

```diff
@@ -0,0 +1,32 @@
+# Technical Fork
```

Comment:
Since the default locale of OpenDigger is `zh`, you copy the file under `i18n/en` folder and add `(To be translated)` in the title of this file.